examples/ssd_detection/test.cpp

```diff
@@ -0,0 +1,420 @@
+  n_confidence_items += conf_feature3.size();
+  concat_hwc_features(confidences, n_confidence_items, conf_feature4, 5 * 5,
+                      6 * N_CLASSES);
+  n_confidence_items += conf_feature4.size();
+  concat_hwc_features(confidences, n_confidence_items, conf_feature5, 3 * 3,
+                      4 * N_CLASSES);
+  n_confidence_items += conf_feature5.size();
+  concat_hwc_features(confidences, n_confidence_items, conf_feature6, 1 * 1,
+                      4 * N_CLASSES);
+  n_confidence_items += conf_feature6.size();
+  // Softmax
+  inline_softmax(confidences);
+
+  // Get class labels for bounding boxes
+  std::vector<int> bounding_box_indexes;
+  std::vector<int> bounding_box_classes;
+  std::vector<float> bounding_box_confidences;
+  for (size_t i = 0; i < N_ANCHORS; ++i) {
+    int max_conf_id = -1;
+    float max_conf  = -1;
+    for (size_t j = 0; j < N_CLASSES; ++j) {
+      float conf = confidences[i * N_CLASSES + j];
+      if (conf > max_conf) {
+        max_conf_id = j;
+        max_conf    = conf;
+      }
+    }
+    if (max_conf_id != BG_CLASS_ID) {
+      bounding_box_indexes.push_back(i);
+      bounding_box_classes.push_back(max_conf_id);
+      bounding_box_confidences.push_back(max_conf);
+    }
+  }
+
+  // Get valid bounding boxes
+  std::vector<tiny_dnn::bounding_box> bounding_box_candidates;
+  for (size_t i = 0; i < bounding_box_indexes.size(); ++i) {
+    int bbox_index = bounding_box_indexes[i];
+
+    tiny_dnn::bounding_box bbox;
+    bbox.x_min = bounding_boxes[bbox_index * 4] * INPUT_SIZE;
+    bbox.y_min = bounding_boxes[bbox_index * 4 + 1] * INPUT_SIZE;
+    bbox.x_max = bounding_boxes[bbox_index * 4 + 2] * INPUT_SIZE;
+    bbox.y_max = bounding_boxes[bbox_index * 4 + 3] * INPUT_SIZE;
+    bbox.score = bounding_box_confidences[i];
+    bounding_box_candidates.push_back(bbox);
+  }
+
+  // Print coordinates of bounding boxes
+  bounding_box_indexes = tiny_dnn::nms(bounding_box_candidates, NMS_THRESHOLD);
+  if (bounding_box_indexes.size()) {
+    std::cout << "Bounding box coordinates:" << std::endl;
+  } else {
+    std::cout << "No targets detected." << std::endl;
+  }
+  for (size_t i = 0; i < bounding_box_indexes.size(); ++i) {
+    int bbox_index              = bounding_box_indexes[i];
+    tiny_dnn::bounding_box bbox = bounding_box_candidates[bbox_index];
+
+    std::cout << "x_min = " << bbox.x_min << ", "
+              << "x_max = " << bbox.x_max << ", "
+              << "y_min = " << bbox.y_min << ", "
+              << "y_max = " << bbox.y_max << ", "
+              << "class = " << bounding_box_classes[bbox_index] << ", "
+              << "score = " << bounding_box_confidences[bbox_index]
+              << std::endl;
+  }
+}
+
+int main(int argc, char** argv) {
+  if (argc != 3) {
+    std::cout << "Usage: example_ssd_test model_folder_path img_file_path";
+    return -1;
+  }
+  std::vector<tiny_dnn::network<tiny_dnn::sequential>> nets;
+
+  construct_nets(nets, argv[1]);
```

Comment:
User specified `model_folder_path` argument may not contain trailing path separator letter, so adding it would be helpful.